web-app/src/app/components/flux/external-data-input/external-data-input.component.ts

```diff
@@ -21,6 +21,8 @@ import { BaseInputComponent } from '../BaseInput.component';
 export class ExternalDataInputComponent extends BaseInputComponent implements OnInit, AfterViewInit {
   @Input() link_name: string = '';
 
+  currentUser = inject(UserService).userCurrent
```

Comment:
Pourquoi récuperer le current user par injection de service:  UserService et non pas en utilisant sharedData :     id_e = this.sharedDataService.getUser().user_info.id_e;